### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by CMUgarte**

### CODE REVIEW

**Review by chrzyki — COMMENTED** [review]
I think Tiago as `Other` here would be better. Keep note about his contributions but he's not a current editor.

File: cldf/cldf-metadata.json [hunk 1]
```diff
@@ -16,25 +16,25 @@
...
             "rdf:type": "prov:Entity",
-            "dc:created": "v2.5.0",
+            "dc:created": "v3.2.0",
             "dc:title": "Concepticon"
         },
         {
             "rdf:about": "https://github.com/cldf-clts/clts",
             "rdf:type": "prov:Entity",
-            "dc:created": "v2.1.0",
+            "dc:created": "v2.2.0",
```
Comment (chrzyki) [inline cldf/cldf-metadata.json:37]: Please build with CLTS v2.3.0.

**Comment by MuffinLinwist** [comment]
@chrzyki I did some further changes and updated the CLTS version to 2.3.0

File: .github/workflows/cldf-validation.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6]
+        python-version: [3.9, "3.10", 3.11, 3.12]
```
Comment (FredericBlum) [inline .github/workflows/cldf-validation.yml:15]: Reduce to one version

**Review by FredericBlum — APPROVED** [review]

**Pull request merged**